garak/resources/gcg/generate_gcg.py

```diff
@@ -37,7 +37,8 @@
 
 logger = getLogger(__name__)
 
-gcg_resource_data = garak._config.transient.basedir / "resources" / "gcg" / "data"
+resource_data = garak._config.transient.basedir / "resources"
+gcg_resource_data = garak._config.transient.cache_dir / "resources" / "gcg" / "data"
```

Comment:
Consider updating `suffix.GCGCached` to read from the cache location and copy from the project install if the file is not already in this cache_dir path.

Reply:
Deferring this idea for [future iteration](https://github.com/leondz/garak/pull/799/files/bdc2a416078522b9dd0052b5036e834f27598a21#r1688563991).